### ISSUE TITLE
Avatar goes missing every now and then on remote nodes

### ISSUE THREAD

**Issue by SpcCw** [friendica/friendica#14316]
I have a weird issue lasting for more than a year now - sometimes users of remote nodes complain that avatars of some users on my node go missing for a while and recover after some days/weeks. At first I blamed connectivity (i.e. when some remote node tries to refresh information perhaps something network-related happens so refresh fails until the next attempt) but recently I noticed something strange.

Yesterday my own avatar went missing on some Mastodon servers for a few days. Everything seemed okay with my account (avatar available, not every server affected) and where remote admins forced refresh my avatar reappeared. However if I checked like this:
`curl -H Accept:application/activity+json https://friends.deko.cloud/profile/shuro | jq '{"icon"}'`
I got no icon. I even re-set my avatar to make sure and nothing changed.
![image](https://github.com/user-attachments/assets/55037216-b84d-4ded-b3e3-3c1fc23f2d50)

Today it returns an avatar in this way (I changed nothing, even the terminal session is the same):
![image](https://github.com/user-attachments/assets/a32d200b-9489-4e59-83b8-f9407f65669f)

I checked some other users and some Friendica users return this, some do not. It seems that ALL users of other platforms with existing avatars return this but with Friendica it is inconsistent. E.g. I took liberty to check accounts of two lead Friendica developers to make sure it is not just me and no icons either:
![image](https://github.com/user-attachments/assets/b7779128-1b8f-4bdc-9c92-a83912d02e8f)

Is this inconsistent behavior normal? Shouldn't it be always returned like on the other platforms?

**Comment by MrPetovan** [comment]
Thank you for looking into this! The curl calls are very helpful.

**Comment by SadmL** [comment]
Yeah, I've encountered with this on my fresh (1+ week) GoToSocial instance, only Friendica users have blank avatars.
Original profile:
![23-08-2024_11-29](https://github.com/user-attachments/assets/54ed4bae-8093-4e77-a727-ca1489890b8f)
What I see
![изображение](https://github.com/user-attachments/assets/9d1662ba-0cfc-4882-b0e0-b16c839eda56)

**Comment by jyrgi66** [comment]
Same here on my GTS Instance. Is no one interested in solving this issue?
